examples/amp-story/performance/lcp-compressed.html

```diff
@@ -0,0 +1,105 @@
+<!-- Use #load=first to enable the experiment. -->
+<!doctype html>
+<html ⚡ lang="en">
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="lcp-compressed.html">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <title>LCP test</title>
+
+  <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
+  <!-- Fonts -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Source+Code+Pro&family=Source+Sans+Pro:wght@400;600&display=swap" rel="stylesheet">
+
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <style amp-custom>
+    * {
+      box-sizing: border-box;
+    }
+    amp-story-page {
+      font-family: 'Source Sans Pro', sans-serif;
+      color: white;
+      background: black;
+    }
+    code {
+      font-family: 'Source Code Pro', monospace;
+      background: #fff2;
+    }
+    amp-story p {
+      line-height: 1.5;
+    }
+    .bold {
+      font-weight: bold;
+    }
+    .bottom {
+      align-content: end;
+    }
+    .last {
+      padding-bottom: 8.83vh;
+    }
+    .byline {
+      letter-spacing: 1.28px;
+    }
+    amp-img {
+      margin: 1em;
+      margin-bottom: 5em;
+    }
+  </style>
+</head>
+  <body>
+    <script>
+      const loadFirst = location.hash.includes('load=first');
+      (self.AMP = self.AMP || []).push(function(AMP) {
+        AMP.toggleExperiment('amp-story-load-first-page-only', loadFirst);
+      });
+    </script>
```

Comment:
@gmajoulet we need to keep the JS code to enable the experiment which will differ from prod, but otherwise not making any HTML changes to the doc here. Instead, I created another example called `lcp-compressed.html` in the folder `amp-story/performance`, where we can add more example stories to test different scenarios